app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -507,12 +507,30 @@
     <string name="no_one_watching">无人在线观看</string>
     <plurals name="watching">
         <item quantity="one">%s 人在观看</item>
-        <item quantity="other"></item>
+        <item quantity="other"/>
     </plurals>
     <string name="no_one_listening">没人在听</string>
     <plurals name="listening">
-        <item quantity="one">％s个听众</item>
+        <item quantity="one">％s 人在听</item>
         <item quantity="other"/>
     </plurals>
     <string name="localization_changes_requires_app_restart">重新启动应用后，语言将更改。</string>
+    <string name="peertube_instance_url_title">PeerTube 服务器</string>
+    <string name="peertube_instance_url_summary">设置自己喜欢的peertube服务器</string>
+    <string name="peertube_instance_url_help">查找最适合你的服务器https://joinpeertube.org/instances#instances-list</string>
```

Comment:
You should use %s (check source strings.xml)

Reply:
source strings in Weblate are so—it also writes the URL directly.

Reply:
https://hosted.weblate.org/translate/newpipe/strings/en/?q=peertube_instance_url_help&checksum=&offset=1 Even on weblate it writes %s instead of the URL

Reply:
well，old string.